Makefile.e2e

```diff
@@ -201,7 +201,7 @@ test-e2e-kind-multi-repo-test-group2:
 
 # This target runs the third group of e2e tests with the multi-repo mode.
 test-e2e-kind-multi-repo-test-group3:
-	$(MAKE) E2E_ARGS="$(E2E_ARGS) --test-features=reconciliation-2" test-e2e-kind-multi-repo
+	$(MAKE) E2E_ARGS="$(E2E_ARGS) --test-features=reconciliation-2,multi-repos,override-api,workload-identity,hydration" test-e2e-kind-multi-repo
```

Comment:
This is not needed as the workload identity tests only run on GKE clusters.

Reply:
I see. Good catch. I didn't noticed it has `ntopts.RequireGKE(t)`.
Removed it.